### PR TITLE
RavenDB-10608 - correctly identify running in a container

### PR DESCRIPTION
**Pull request by grisha-kotler**
http://issues.hibernatingrhinos.com/issue/RavenDB-10608

### CODE REVIEW

File: src/Sparrow/LowMemory/MemoryInformation.cs [hunk 1]
```diff
@@ -165,7 +164,7 @@ public static long GetRssMemoryUsage(int processId)
             }
         }
 
-        public static (long MemAvailableInKb, long SwapTotalInKb, long CommitedInKb, long TotalMemoryInKb) GetFromProcMemInfo()
+        public static (long MemAvailable, long SwapTotal, long Commited, long TotalMemory) GetFromProcMemInfo()
```
Comment (arekpalinski) [inline src/Sparrow/LowMemory/MemoryInformation.cs:167]: Why dropping unit suffix? Can we use `Size` struct here instead?

Comment (grisha-kotler) [reply src/Sparrow/LowMemory/MemoryInformation.cs:167]: The caller works with bytes

Comment (arekpalinski) [reply src/Sparrow/LowMemory/MemoryInformation.cs:167]: You know that but someone changing this in the future code will need to figure it out. Putting this as `MemAvailableInBytes` will make it clearer. Using `Size` here would make it more flexible regardless the unit that the caller works with

Comment (grisha-kotler) [reply src/Sparrow/LowMemory/MemoryInformation.cs:167]: Fixed

Comment (arekpalinski) [reply src/Sparrow/LowMemory/MemoryInformation.cs:167]: Great. Thanks

**Pull request merged**